app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java

```diff
@@ -33,6 +33,7 @@
     private SwipeRefreshLayout swipeContainer;
 
     private SubmissionAdapter submissionAdapter;
+    private static final String url = "url";
```

Comment:
all the constants or final variables should be in capitals like `final String URL = `